go.mod

```diff
@@ -257,7 +257,7 @@ replace (
 	// use cosmos flavored protobufs
 	github.com/gogo/protobuf => github.com/regen-network/protobuf v1.3.3-alpha.regen.1
 
-	github.com/realio-tech/multi-staking-module => github.com/decentrio/multi-staking v0.0.0-20241025035539-29cf83199609
+	github.com/realio-tech/multi-staking-module => github.com/DongLieu/multi-staking v0.0.0-20250123104107-066370befd7b
```

Comment:
Can you explain this change here?

Reply:
It's just for testing, we should close this pr now